Gemfile.lock

```diff
@@ -70,10 +72,13 @@ GEM
     ci_reporter (1.7.0)
       builder (>= 2.1.2)
     cliver (0.3.2)
+    coderay (1.1.0)
+    columnize (0.9.0)
     connection_pool (1.2.0)
     crack (0.4.2)
       safe_yaml (~> 1.0.0)
-    database_cleaner (0.7.2)
+    database_cleaner (1.4.1)
+    debugger-linecache (1.2.0)
     devise (3.1.0)
       bcrypt-ruby (~> 3.0)
```

Comment:
bcrypt is a devise dependency, we don't list it in the Gemfile or use it directly in the app…